### PR TITLE
Feature/#20 mobile hero

### PR DESCRIPTION
**Pull request by dutraneto**
Hey, @marceloglacial #20 (Hero section) ready for review.

### CODE REVIEW

**Review by marceloglacial — CHANGES_REQUESTED** [review]
Good job. 
Two things to fix:

1) Header heigh is 100vh and overlapping the content (I missed the on the other review)
Screen Shot 2021-03-05 at 3 15 00 PM

2) Filter props has a few knowing issues, so try to avoid it. (https://caniuse.com/?search=filter). You can use a pseudo element (:before) to create the brightness effect (another thing that I missed from the laters reviews. Sorry about that)

Screen Shot 2021-03-05 at 3 15 36 PM

**Comment by dutraneto** [comment]
> Good job.
> Two things to fix:
> 
> 1. Header heigh is 100vh and overlapping the content (I missed the on the other review)
> 
> Screen Shot 2021-03-05 at 3 15 00 PM
> 
> 1. Filter props has a few knowing issues, so try to avoid it. (https://caniuse.com/?search=filter). You can use a pseudo element (:before) to create the brightness effect (another thing that I missed from the laters reviews. Sorry about that)
> 
> Screen Shot 2021-03-05 at 3 15 36 PM

1. It was a good catch. I was supposed to fix that and ended up forgetting it.
2. Hey, reviewing the design I just realized that filter value was removed since the last time you updated it and applied a bg-color to title. So, I've just removed it, too =).

image

**Comment by dutraneto** [comment]
#20 ready after changes were made. Please, review it as soon as possible. Thank you!

**Review by marceloglacial — APPROVED** [review]

File: src/styles/components/_hero.scss [hunk 1]
```diff
@@ -7,12 +7,15 @@
...
+
+        @include media('<=tablet') {
+            margin-left: 0;
+        }
     }
 
     &__img {
         height: 100%;
         object-fit: cover;
-        filter: brightness(60%);
```
Comment (marceloglacial) [inline src/styles/components/_hero.scss:15]: Where's the alternative code? 🤔

Comment (marceloglacial) [reply src/styles/components/_hero.scss:15]: Nevermind. I forgot I removed the background on the prototype hahaha Sorry. My bad :)

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**